modules/ols-byok-task.adoc

```diff
@@ -0,0 +1,113 @@
+
+[NOTE]
+====
+Adding custom knowledge to the LLM is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
+
+* You have configured the large language model (LLM) provider.
+
+* You have installed the {ols-long} Operator.
+
+* The custom information you want to add resides in a Markdown file; no other file format is supported.
+
+* You have an account for a container image registry, such as `quay.io`.
+
+.Procedure 
+
+. Specify the location of the directory with the markdown files for the custom RAG database and the path for the image that the BYO Knowledge tool generates by running the following command: 
++
+[source,terminal]
+----
+$ podman run -e OUT_IMAGE_TAG=my-byok-image -it --rm --device=/dev/fuse -v <directory_for_markdown_files>:/markdown:Z -v <directory_for_image_tar>:/output:Z $MY_BYOK_TOOL_IMAGE
+----
++
+[NOTE]
+====
+The container image that the tool generates contains a custom RAG database. The RAG database provides additional information to the LLM. 
+====
+
+. Load the container image that the BYO Knowledge tool generated by running the following command:
++
+[source,terminal]
+----
+$ podman load < <directory_for_image_tar>/<my-byok-image.tar>
+----
+
+. Display the podman images that reside on your laptop by running the following command:
++
+[source,terminal]
+----
+$ podman images
+----
++
+.Example output
+[source,terminal]
+----
+REPOSITORY                            TAG                IMAGE ID      CREATED       SIZE
+localhost/my-byok-image               latest             be7d1770bf10  3 days ago    2.37 GB
+...
+----
+
+. Tag the local image with a new name and destination so that the image can be pushed to the container image registry by running the following command:
++
+[source,terminal]
+----
+$ podman tag localhost/my-byok-image:latest quay.io/<username>/my-byok-image:latest
+----
+
+. Push the local container image to the container image registry by running the following command:
++
+[source,terminal]
+----
+$ podman push quay.io/<username>/my-byok-image:latest
+----
+
+. Modify the `OLSconfig` custom resource file to deploy the newly created RAG database alongside the existing one:
```

Comment:
custom resource file -> custom resource
since we're editing the CR in the web console

Reply:
done